lib/templates/deploy.batch.dnx.consoleapp.template

```diff
@@ -0,0 +1,21 @@
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+:: Deployment
+:: ----------
+
+echo Handling DNX Console Application deployment.
+
+:: 1. Set WebJob deployment target
+IF NOT DEFINED WEBJOB_DEPLOYMENT_TARGET (
+  SET WEBJOB_DEPLOYMENT_TARGET=%DEPLOYMENT_TARGET%\App_Data\jobs\continuous\deployedJob
+)
+
+:: 2. Create WebJob directory
+IF EXIST "WEBJOB_DEPLOYMENT_TARGET" (
+  CALL :ExecuteCmd mkdir "%WEBJOB_DEPLOYMENT_TARGET%"
+)
+
+:: 3. KuduSync
+CALL :ExecuteCmd "%KUDU_SYNC_CMD%" -v 50 -f "{WEBJOB_PROJECT_PATH}" -t "%WEBJOB_DEPLOYMENT_TARGET%" -n "%NEXT_MANIFEST_PATH%" -p "%PREVIOUS_MANIFEST_PATH%" -i ".git;.hg;.deployment;deploy.cmd"
```

Comment:
I see. So maybe I'll do the same thing as the regular .NET console app deployment script where msbuild outputs all the files to a temp folder with the right structure (i.e: `temp\App_Data\jobs\con....`) and then copies from that to wwwroot. I thought since there is no build step here we can bypass that and go straight from the repo